app/helpers/application_helper.rb

```diff
@@ -573,6 +573,16 @@ def prefix_properties(concept_properties)
     modified_properties
   end
 
+  def convert_dates(hash)
```

Comment:
put it in the `concept_details_component.rb`

Reply:
done